ensembl/htdocs/info/genome/compara/species_trees.html

```diff
@@ -17,19 +17,72 @@ <h1 id="speciestrees">Species trees</h1>
 <ol>
     <li>The <a href="https://www.ncbi.nlm.nih.gov/taxonomy">NCBI taxonomy</a> (i.e. topology only) in:
```

Comment:
The reference to non-vertebrate divisions could be moved here...
```suggestion
    <li>The <a href="https://www.ncbi.nlm.nih.gov/taxonomy">NCBI taxonomy</a> (i.e. topology only) is typically used for non-vertebrate divisions in:
```